core/parentchain/indirect-calls-executor/Cargo.toml

```
@@ -17,13 +17,16 @@ itp-ocall-api = { path = "../../../core-primitives/ocall-api", default-features
 itp-settings = { path = "../../../core-primitives/settings" }
 itp-sgx-crypto = { path = "../../../core-primitives/sgx/crypto", default-features = false }
 itp-stf-executor = { path = "../../../core-primitives/stf-executor", default-features = false }
+itp-top-pool-author = { path = "../../../core-primitives/top-pool-author", default-features = false }
 itp-types = { path = "../../../core-primitives/types", default-features = false }
 
 # sgx enabled external libraries
 thiserror_sgx = { package = "thiserror", git = "https://github.com/mesalock-linux/thiserror-sgx", tag = "sgx_1.1.3", optional = true }
+futures_sgx = { package = "futures", git = "https://github.com/mesalock-linux/futures-rs-sgx", optional = true }
 
 # std compatible external libraries (make sure these versions match with the sgx-enabled ones above)
 thiserror = { version = "1.0", optional = true }
+futures = { version = "0.3.8", default-features = false, optional = true }
```

```suggestion
futures = { version = "0.3.8", optional = true }
```

Why are you adding`default-features = false` if you're adding `std` in the `feature` anyway?

---

hm yes, that is a leftover from my try to use that library in `sgx` as well 👍